geometry_msgs/Point[] goal_configurations
---
bool success
string message
